jupyterlab_bigquery/src/utils/widgetManager/widget_manager.ts

```diff
@@ -0,0 +1,98 @@
+import { ReactWidget, MainAreaWidget } from '@jupyterlab/apputils';
+import { JupyterFrontEnd } from '@jupyterlab/application';
+import { configureStore, EnhancedStore } from '@reduxjs/toolkit';
+import rootReducer from '../../reducers';
+import { Widget } from '@phosphor/widgets';
+import { ReduxReactWidget } from './redux_react_widget';
+
+/**
+ * A class that manages dataset widget instances in the Main area
+ */
+export class WidgetManager {
```

Comment:
Can you sync this with the upstream? This file was refactored in https://github.com/GoogleCloudPlatform/jupyter-extensions/pull/83

Reply:
Done